cpt/runner.py

```diff
@@ -29,7 +29,6 @@ def __init__(self, profile_abs_path, reference, conan_api, uploader,
         self._exclude_vcvars_precommand = exclude_vcvars_precommand
         self._build_policy = build_policy
         self._runner = PrintRunner(runner or os.system, self.printer)
-        self._uploader.remote_manager.add_remotes_to_conan()
```

Comment:
I think this is going to break something when running docker. Have you tried?

Reply:
No, I haven't. Also, currently I am basically in the middle of nowhere, so I can't test it now.

Reply:
Indeed, it will. I see these ways to fix this:
1. Add remotes both in packager's and in runner's constructors. This will print a message about remote already existing for each remote if not running jobs in Docker.
2. Same as 1, but add a flag to `RemotesManager`, which is set and checked in `add_remotes_to_conan`.
3. Add remotes in `RemotesManager`'s constructor.
4. Add remotes in packager's constructor and in `cpt.run_in_docker:run`.

What is the most desired approach? Personally, I like approach 2 the most.